polkadot/xcm/xcm-builder/src/unique_instances/adapter.rs

```diff
@@ -0,0 +1,150 @@
+use core::marker::PhantomData;
```

Comment:
@mrshiposha please also check all other files, they are missing headers (licenses and stuff)